exercises/decimal/example.rs

```diff
@@ -117,7 +118,7 @@ macro_rules! auto_impl_decimal_ops {
 
 auto_impl_decimal_ops!(Add, add, |s, o| s + o, |s, _| s);
 auto_impl_decimal_ops!(Sub, sub, |s, o| s - o, |s, _| s);
-auto_impl_decimal_ops!(Mul, mul, |s, o| s * o, |s, o| s + o);
+auto_impl_decimal_ops!(#[allow(clippy::suspicious_arithmetic_impl)] Mul, mul, |s, o| s * o, |s, o| s + o);
```

Comment:
upon review, since the attribute goes on the `mul`, perhaps it would have been better to write this line as:

```rust
 auto_impl_decimal_ops!(Mul, #[allow(clippy::suspicious_arithmetic_impl)] mul, |s, o| s * o, |s, o| s + o);
```

(with the corresponding change in the macro)

I guess it can be revisited if there's ever a need to have an attribute on both the `impl ` and the `fn`.